framework/messages/el/yii.php

```diff
@@ -114,4 +83,32 @@
     '{attribute} should contain at least {min, number} {min, plural, one{character} other{characters}}.' => 'Το «{attribute}» πρέπει να περιέχει το λιγότερο {min, number} {min, plural, one{χαρακτήρα} few{χαρακτήρες} many{χαρακτήρες} other{χαρακτήρες}}.',
     '{attribute} should contain at most {max, number} {max, plural, one{character} other{characters}}.' => 'Το «{attribute}» πρέπει να περιέχει το πολύ {max, number} {max, plural, one{χαρακτήρα} few{χαρακτήρες} many{χαρακτήρες} other{χαρακτήρες}}.',
     '{attribute} should contain {length, number} {length, plural, one{character} other{characters}}.' => 'Το «{attribute}» πρέπει να περιέχει {length, number} {length, plural, one{χαρακτήρα} few{χαρακτήρες} many{χαρακτήρες} other{χαρακτήρες}}.',
+    '{delta, plural, =1{a day} other{# days}} ago' => 'πριν {delta, plural, =1{μία ημέρα} other{# ημέρες}}',
+    '{delta, plural, =1{a minute} other{# minutes}} ago' => 'πριν {delta, plural, =1{ένα λεπτό} other{# λεπτά}}',
+    '{delta, plural, =1{a month} other{# months}} ago' => 'πριν {delta, plural, =1{ένα μήνα} other{# μήνες}}',
+    '{delta, plural, =1{a second} other{# seconds}} ago' => 'πριν {delta, plural, =1{ένα δευτερόλεπτο} other{# δευτερόλεπτα}}',
+    '{delta, plural, =1{a year} other{# years}} ago' => 'πριν {delta, plural, =1{ένα έτος} other{# έτη}}',
+    '{delta, plural, =1{an hour} other{# hours}} ago' => 'πριν {delta, plural, =1{μία ώρα} other{# ώρες}}',
+    '{nFormatted} B' => '{nFormatted} B',
+    '{nFormatted} GB' => '{nFormatted} GB',
+    '{nFormatted} GiB' => '{nFormatted} GiB',
+    '{nFormatted} KB' => '{nFormatted} KB',
+    '{nFormatted} KiB' => '{nFormatted} KiB',
+    '{nFormatted} MB' => '{nFormatted} MB',
+    '{nFormatted} MiB' => '{nFormatted} MiB',
+    '{nFormatted} PB' => '{nFormatted} PB',
+    '{nFormatted} PiB' => '{nFormatted} PiB',
+    '{nFormatted} TB' => '{nFormatted} TB',
+    '{nFormatted} TiB' => '{nFormatted} TiB',
+    '{nFormatted} {n, plural, =1{byte} other{bytes}}' => '{nFormatted} {n, plural, =1{byte} other{bytes}}',
+    '{nFormatted} {n, plural, =1{gibibyte} other{gibibytes}}' => '{nFormatted} {n, plural, =1{gibibyte} other{gibibytes}}',
```

Comment:
Which words from the quoted lines should be translated? If you're referring to the measurement units (bytes, kilobytes, etc.) I leave them as is, they do not have a corresponding Greek term.